kubernetes/kube-state-metrics-role-binding.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v1beta2
```

Comment:
This should not be changed to `v1beta2` as `Role` and `RoleBinding` has moved to `v1` along side `ClusterRoleBinding` and `ClusterRole`.

@zouyee